annotator/src/main/java/org/cbioportal/models/AnnotatedRecord.java

```diff
@@ -58,7 +59,8 @@
         HEADER.add("Protein_position");
         HEADER.add("Codons");        
         HEADER.add("Hotspot");
-    }       
+        HEADER.add(HEADER.indexOf("Variant_Classification"), "Consequence");
```

Comment:
Add does not replace the element, it sticks in at that position and moves all other elements back one index.